### DISCUSSION TITLE
Error referencing `environ` with Linux SDK

### DISCUSSION THREAD

**Discussion by seanmrich** [hummingbird-project/hummingbird#563] [Q&A]
I created a test project using the template and tried cross compiling for Linux using the latest release versions of swift and the static SDK. I get an error that `environ` is not concurrency-safe in Sources/Hummingbird/Environment line 111.

```bash
% git clone https://github.com/hummingbird-project/template
% ./template/configure.sh Hummer
% cd Hummer
% swift build --swift-sdk x86_64-swift-linux-musl
/.../Hummer/.build/checkouts/hummingbird/Sources/Hummingbird/Environment.swift:111:26: error: reference to var 'environ' is not concurrency-safe because it involves shared mutable state
```

I tried changing the package's tools version to swift 5.9, but got the same result. The `environ` variable is defined in `Darwin` so I'm not sure how to proceed here. It compiles fun under macOS but not Linux.

swift toolchain org.swift.6010202409231a (version 6.0.1)
linux sdk swift-6.0.1-RELEASE_static-linux-0.0.1
hummingbird 2.0.1

**Comment by adam-fowler** [comment]
This was a warning beforehand, and we could ignore it. It is really an issue with the implementation in the static SDK. I was going to add an issue for this but I just haven't managed to get around to it. If you could do so and link back to it here that would be really helpful. I'm away on holiday for 2 weeks so can't really do it just now.
